lib/styles/v2/components/_tag.scss

```
@@ -0,0 +1,43 @@
+$solid-indigo: #5E5CE6;
+$solid-indigo-bg: #EAF3FC;
+
+.nui-tag {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  color: theme("colors.gray.800");
```

```suggestion
  color: $gray-800;
```

We shouldn't use tailwind themes in v2.